### ISSUE TITLE
Add web sockets for control

### ISSUE THREAD

**Issue by dtex** [Houston-Robotics-Club/ariel#3]
Set up socket server and client to handle robot control using [socket.io](http://socket.io/)

It may be possible to do messaging via RTC as well, but web sockets are another good thing for everyone to get their hands on.

**Comment by kwhitsett** [comment]
I can help work on this after i get back from vacation. Can you get Abhinav to put you on GroupMe so that we can all talk about things in the same place?
Sent from Yahoo Mail for iPadAt Jun 27, 2015, 10:18:20 PM

**Comment by kwhitsett** [comment]
Let me rephrase.  This is the appropriate place for doing what you're doing right now, but Groupme is better for back and forth chatting about stuff.

**Comment by egrliz** [comment]
I am concerned about Groupme. It asked for login with Facebook which is fine. Then it asked for my password which is unnecessary. If they used the proper Facebook plugins, then requiring users to enter a password would be redundant. It is easy to write scripts to snatch the password as you type it. Since my facebook was hacked in the past, I would prefer another platform to Groupme.

**Comment by dtex** [comment]
I've added the web sockets layer
